components/mosart/src/inundation/MOSARTinund_Core_MOD.F90

```diff
@@ -135,8 +135,7 @@ subroutine inund_hillslopeRouting ( )
     integer :: iu           ! Computation unit index.
     real( r8 ) :: hsV       ! Hillslope flow velocity (m/s).
       
-    !$OMP PARALLEL FIRSTPRIVATE( iu, hsV )
-    !$OMP DO SCHEDULE( GUIDED )
+    !$OMP PARALLEL DO PRIVATE(hsV) SCHEDULE(GUIDED)
```

Comment:
@amametjanov Just curious, these changes are needed for building with xlf or ...

Reply:
Yes, xlf issues a build error if loop index var is inside firstprivate clause.